packages/flutter/lib/src/material/app.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:ui' show Window; // for dartdocs
```

Comment:
i'd rather we didn't go down this route and just waited until we could do this actually at the dartdoc level.